scripts/import-update-key

```diff
@@ -10,11 +10,14 @@ if [ -z "$keyfile" ]; then
   exit 1
 fi
 
-keyhash=$(gpg $gpgopts --with-fingerprint $keyfile | sed -ne 's#^pub  [^ ]\+/\([^ ]\+\).*#\1#p')
-if [ ${#keyhash} -ne 8 ]; then
+keyid=$(gpg $gpgopts --list-packets --keyid-format LONG $keyfile | grep -Po 'keyid:\s\K.*')
```

Comment:
This will find 2 lines for a key that has signatures. E.g. you can test with my key https://keyserver.pgp.com/vkd/DownloadKey.event?keyid=0x814A3A7A06AA1295

It prints
```
gpg --list-packets --keyid-format LONG mine|grep -Po 'keyid:\s\K.*'
814A3A7A06AA1295
2506AB33A853DC7A
```

Reply:
Or this which doesn't show that warning:
```
gpg --with-fingerprint --with-colons --show-keys --batch mine
pub:e:4096:1:814A3A7A06AA1295:1339529952:1497209952::-:::sc::::::23::0:
fpr:::::::::B32F9BE2E22D4615ABD2CE8E814A3A7A06AA1295:
uid:e::::1369766480::480A52AC965170D5494CFBE73856403937635882::Török Edwin <edwin@skylable.com>::::::::::0:
sub:e:4096:1:2506AB33A853DC7A:1339529952:1497209952:::::e::::::23:
fpr:::::::::C3ECE7505A9BE81FDD7BF3DE2506AB33A853DC7A:
```

Reply:
updated to use `--show-keys` and fallback to `--with-fingerprint`